app/assets/stylesheets/refinery/admin/tags.scss

```diff
@@ -18,21 +26,31 @@
 .tags__header-topic {
   color: #111436;
   font-weight: bold;
-  font-size: .9rem;
+  font-size: .8125rem;
 }
 
 .tags__selector-topic {
-  background-color: none;
-  border: .1rem solid #f0efe7;
+  background-color: #62bef2;
   border-radius: .3rem;
-  font-size: .9rem;
+  box-shadow: 1px 1px 1px black;
+  display: inline-block;
   margin-bottom: .1rem;
-  padding-right: 1rem;
-  text-align: center;
-  width: 7rem;
+  padding: .5rem;
+  white-space: nowrap;
 }
 
 .tags__selector-topic:hover {
-  background-color: #a4bbeb;
   color: #fff;
 }
+
+.field {
+  margin: 0 0 .5rem 0;
+}
+
+.nothing_selected,
```

Comment:
This is very nitpicky, but I'd recommend changing the word separators from underscores to hyphens

Reply:
This particular CSS class was gifted to us by refinery so I don't think I am able to do that for this one. Otherwise I would normally agree!

Reply:
Ohhh weird, no worries then.